src/style/_layout.scss

```
@@ -500,6 +500,17 @@
         {
             font-weight: bold;
         }
+
+        button
```

Can I suggest one additional change?
```suggestion
        button.tablinks
```

---

Done 👍